tests/unit/test_textInfos.py

```diff
@@ -0,0 +1,131 @@
+﻿# -*- coding: UTF-8 -*-
```

Comment:
Per the Contributing guide, please remove the UTF-8 BOM. This won't break here because xgettext isn't used on unit tests, but best to be consistent.